src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -91,11 +93,19 @@ public static function getSubscribedEvents()
     protected function logException(\Exception $exception, $message)
     {
         if (null !== $this->logger) {
-            if (!$exception instanceof HttpExceptionInterface || $exception->getStatusCode() >= 500) {
-                $this->logger->critical($message, array('exception' => $exception));
-            } else {
-                $this->logger->error($message, array('exception' => $exception));
+            $logLevel = LogLevel::ERROR;
+            if ($exception instanceof HttpExceptionInterface) {
+                $statusCode = $exception->getStatusCode();
+                if (isset(static::$httpStatusCodeLogLevel[$statusCode])) {
+                    $logLevel = static::$httpStatusCodeLogLevel[$statusCode];
+                } else if ($statusCode >= 500) {
+                    $logLevel = LogLevel::CRITICAL;
+                } else if ($statusCode >= 400) {
+                    $logLevel = LogLevel::WARNING;
```

Comment:
previously "error".. but i like this :+1:

Reply:
Yes you're right, this is a change. WARNING seems a much more logical logging level for 4xx to me.

Reply:
^ thoughts @nicolas-grekas ?

Reply:
I've restored the levels to what they originally were, as I'd prefer discussion to be merits of the overall change rather than the individual levels. We could follow up with a separate PR if you think it does make sense to de-emphasise http 4xx